docs/sources/panels/transformations/apply.md

```diff
@@ -0,0 +1,32 @@
++++
+title = "Apply a Transformation"
+type = "docs"
+[menu.docs]
+identifier = "apply_transformation"
+parent = "tranformations"
+weight = 300
++++
+
+
+# Apply a transformation
```

Comment:
File names should match the topic names as closely as possible. In this case, apply-a-transformation.md. Apply throughout.

Reply:
Updated.